homeworks/08-functional-reactive-programming/src/components/Grid.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Grid.css';
+
+import { Cat } from './Cat';
+import { Wall } from './Wall';
+import { Window } from './Window';
+
+import { Cell, CellContent } from '../types';
+
+interface GridProps {
+  cells: Cell[],
+  onClick: (content: CellContent) => void,
+}
+
+export const Grid: React.FC<GridProps> = ({ cells, onClick }) => {
+  // eslint-disable-next-line no-undef
+  const gridCells: JSX.Element[] = [];
+  cells.forEach((cell) => {
+    let el = <Wall/>;
```

Comment:
Small refactoring makes this part of your code a bit better.
```typescript
const getCellConmonent = (cell: Cell) => {
  switch (cell.content) {
    case CellContent.CAT:
      return <Cat/>;
    case CellContent.WINDOW:
      return <Window/>;
    default:
      return <Wall/>;
  }
}
```

Reply:
Done in https://github.com/vbarbasins/evo-ts-bootcamp/pull/9/commits/8886c14d4ab012921b8cb9cbebcde5ba235700ec